packages/pigeon/README.md

```diff
@@ -8,24 +8,39 @@ host platform type-safe and easier.
 
 ## Supported Platforms
 
-Currently Pigeon only supports generating Objective-C code for usage on iOS and calling host functions from Flutter.
+Currently Pigeon only supports generating Objective-C code for usage on iOS and
+Java code for Android.
 
 ## Runtime Requirements
 
-Pigeon generates all the code that is needed to communicate between Flutter and the host platform, there is no extra runtime requirement.  A plugin author doesn't need to worry about conflicting versions of Pigeon.
+Pigeon generates all the code that is needed to communicate between Flutter and
+the host platform, there is no extra runtime requirement.  A plugin author
+doesn't need to worry about conflicting versions of Pigeon.
 
 ## Usage
 
-### Steps
+### Flutter calling into iOS Steps
 
 1) Add Pigeon as a dev_dependency.
 1) Make a ".dart" file outside of your "lib" directory for defining the communication interface.
 1) Run pigeon on your ".dart" file to generate the required Dart and Objective-C code.
```

Comment:
This is implying that you should put all your data types (possibly 10 class defs) inside one file right? If so, add that recommendation/restriction, or should pigeon be able to run against a directory?

Reply:
People could split out have pigeon files if they wanted.  You'll just have to make multiple invocations to pigeon.  I don't see a strong case for supporting directories.

Reply:
You can divide it up any way you want.  Nothing stops you from having multiple pigeon files.  I haven't seen a strong case for directory support since shells do a nicer job, it would just complicate things.